### ISSUE TITLE
cleanup gds directory

### ISSUE THREAD

**Issue by proppy** [efabless/caravel_user_project#131]
Is it intentional that the `gds/` directory already contains the hardened project + wrapper?

This could be confusing for developers who expect that the test and mpmprecheck would fails until they actually go thru the step of hardening their design.

**Comment by SaraEfabless** [comment]
we purposely want the GDS there, so, ppl dont have to generate it to see the design or to run check LVS, maybe in the future to have a GDS inside the repo.

**Comment by mattvenn** [comment]
Yes I just got hit by this myself. I think especially now we are supporting multiple PDKs, these binaries should be deleted.

**Comment by mattvenn** [comment]
@SaraEfabless it's kind of difficult to just run LVS on its own. IMO it's better if people build the GDS, then they can check all the results and know that the artifacts are generated by the current tools and pdk.

**Comment by proppy** [comment]
alternatively, the github action could generate the gds for `sky130` and `gf180mcu` and commit them to a different branch?

That would be assuming that the efabless platform is capable of fetching a different branch than the default one for a given tapeout.

**Comment by mattvenn** [comment]
I think having a different branch specified would be good. Then it could be possible to have one repo for gf180 and sky130
